docs/client/README.md

```diff
@@ -15,11 +15,19 @@ See [Install and Set Up `kubectl`](https://kubernetes.io/docs/tasks/tools/instal
 !!! note
     `kn` cannot be used to install Knative components such as Serving or Eventing.
 
-See [Installing `kn`](install-kn.md).
+### Additional resources
 
-### kn reference documentation
+- See [Installing `kn`](install-kn.md).
+- See the [`kn` documentation]({{ clientdocs() }}){target=_blank} in Github.
 
-See the [`kn` documentation]({{ clientdocs() }}){target=_blank} in Github.
+## func
+
+The `func` CLI enables you to create, build, and deploy Knative Functions without the need to create or modify YAML files directly.
+
+### Additional resources
+
+- See [Installing Knative Functions](../functions/install-func.md).
+- See the [`func` documentation]({{ funcdocs() }}){target=_blank} in Github.
```

Comment:
This `funcdocs()` function should be defined in `hack/macros.py`. See https://github.com/knative/docs/pull/5243/files#diff-3b13ac291b4c25e5bc68a1a197012e53854f02dcd29f5f7f1c952f0cd2980656

Reply:
ACK, thanks, I meant to add this and forgot 🤦🏻